tests/PHPStan/Reflection/PhpDefect/PhpDefectClassReflectionExtensionTest.php

```diff
@@ -594,7 +593,6 @@ public function dataLibXMLErrorProperties(): array
 
 	/**
 	 * @dataProvider dataDateInterval71Properties
-	 *
```

Comment:
Removed because of [`SlevomatCodingStandard.Commenting.DocCommentSpacing`](https://github.com/slevomat/coding-standard#slevomatcodingstandardcommentingdoccommentspacing-)

Reply:
I think the original version was correct, so rather the sniff should be configured differently?